src/h5web/vis-packs/core/heatmap/utils.ts

```diff
@@ -2,20 +2,34 @@ import { range } from 'lodash-es';
 import type ndarray from 'ndarray';
 import type { D3Interpolator, Dims } from './models';
 
+const GRADIENT_PRECISION = 1 / 20;
```

Comment:
I'm doubling the precision (was `0.1`). It was a bit too low for cyclical interpolators with lots of colours, like `Rainbow`.